prow/crier/controller.go

```diff
@@ -199,9 +208,14 @@ func (r *reconciler) reconcile(ctx context.Context, log *logrus.Entry, req recon
 		return requeue, nil
 	}
 
-	log.Info("Reported job(s), now will update pj(s)")
+	log.Infof("Reported job(s), now will update %d pj(s)", len(pjs))
```

Comment:
Please use a log field rather than a `printf`-style message so ppl can filter for this log line

Reply:
Done.